source/computer/commands.rst

```diff
@@ -66,51 +66,695 @@ Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这
         - `ssh`_
         - `scp`_
         - `rsync`_
+        - `wget`_
```

Comment:
Need to add metadata at the top of the file.

Reply:
???

Reply:
authors, last updated time, and reading time,

Reply:
done in 4e1ecf6.